movement/log_config.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from logging.handlers import RotatingFileHandler
+
+from rich.logging import RichHandler
+
+FORMAT = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+
+def configure_logging():
+    """Configure the logging module."""
+
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+
+    log_file = "movement.log"
```

Comment:
Presumably this should be in something like `~/.movement/movement/log`? Otherwise is it just in cwd or the package dir (neither desirable).

Reply:
Good point. I am not sending the log to `~/.movement`. Currently, this will also be done on windows (where the folder is not hidden). Could keep it as is, or could move it to the `APPDIR` if on windows.

Reply:
Not sure about this. `~/.movement` isn't hidden on Windows, but plenty of other packages store data there. I'm not a fan of using `APPDIR` for something like this. It may not be very easy for the user to find and I'm not sure if a user will have write permissions to this directory if there's a shared installation of `movement`.

Reply:
OK. Will keep things as they are for now, i.e. in `~/.movement`